app/sql/bakeryDB.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS product (
+    id          UUID          NOT NULL PRIMARY KEY,
+    name        VARCHAR(100)  NOT NULL,
+    quantity    INTEGER,
+    price       DECIMAL,
+    created_at  TIMESTAMP,
+    updated_at  TIMESTAMP
+);
```

Comment:
notice that since you created the table in the `conf/evolutions/default` directory this isn't needed nor is it being used now. Remove

Reply:
removed file and directory